doc/man3/flux_msg_create.rst

```diff
@@ -0,0 +1,79 @@
+==========================
+flux_msg_create(3)
+==========================
+
+.. default-domain:: c
+
+SYNOPSIS
+========
+
+.. code-block:: c
+
+   #include <flux/core.h>
+
+   flux_msg_t *flux_msg_create (int type)
+
+   flux_msg_t *flux_msg_copy (const flux_msg_t *msg, bool payload)
+
+   const flux_msg_t *flux_msg_incref (const flux_msg_t *const_msg)
+
+   void flux_msg_decref (const flux_msg_t *const_msg)
+
+   void flux_msg_destroy (flux_msg_t *msg)
+
+
+DESCRIPTION
+===========
+
+:func:`flux_msg_create` creates a :type:`flux_msg_t` of the :var:`type`.
+Different types of Flux messages are defined in RFC :doc:`rfc:spec_3`. All
+messages have a starting reference count of 1.
+
+:func:`flux_msg_copy` duplicates :var:`msg`. The payload is omitted unless
+:var:`payload` is true. The initial reference count of the new message is 1.
+
+:func:`flux_msg_incref` increments the reference count of :var:`const_msg`
+by 1.
+
+:func:`flux_msg_decref` decrements the reference count of :var:`const_msg`
+by 1. When the reference count reaches 0, the message is destroyed.
+
+:func:`flux_msg_destroy` destroys :var:`msg`.
```

Comment:
`flux_msg_destroy` should just say that it's an alias for :func:`flux_msg_decref`.   They are exactly the same - the message is only destroyed once the reference count reaches zero.